jquery/textSelection.d.ts

```diff
@@ -129,7 +119,55 @@ declare global {
          * @param {string} command Command to execute
          */
         textSelection(command: "unregister"): void;
+
+        /**
+         * Do things to the selection in the textarea, using a command from the alternative textSelection API for this element.
+         *
+         * @param {string} command Command to execute
+         * @param {Mixed} [commandOptions] Options to pass to the command
+         * @return {Mixed} Depending on the command
+         */
+        textSelection(command: string, commandOptions?: any): void;
```

Comment:
The goal of this default case is to allow using commands from an alternative API, registered using the `register` command There may be a better way to do it since this way it basically just disables command options type checking.